src/main/java/edu/unc/lib/boxc/migration/cdm/status/SourceFilesSummaryService.java

```diff
@@ -28,15 +25,12 @@
  */
 public class SourceFilesSummaryService extends AbstractStatusService {
```

Comment:
One last suggestion, we should try to limit which methods are `public` to just the ones intended to be the public API for this class. I believe that means `summary`, `capturePreviousState`, and the getters/setters (and others if you think they should be accessible elsewhere). It helps make it clearer how the class is intended to be used, and helps avoid invalid/unsafe usages. We can make the others private for now, or protected if they are being used directly by the test or subclasses.